### ISSUE TITLE
`build_fuzzers --engine centipede` isn't compatible with local build directories

### ISSUE THREAD

**Issue by evverx** [google/oss-fuzz#9450]
`helper.py` doesn't clean up build directories: https://github.com/google/oss-fuzz/issues/9249 so if build scripts don't do it themselves depending on the build system they can end up failing or reusing unsanitized builds when `helper.py` starts building sanitized builds for centipede. Even though the underlying issue is the same as https://github.com/google/oss-fuzz/issues/9249 I think this issue is different in that `helper.py` doesn't have to be run a few times in a row explicitly.

It's possible to get it around by cleaning up build directories explicitly or by switching to `podman` and applying the patch adding `:O` from https://github.com/google/oss-fuzz/issues/9249.

**Comment by jonathanmetzman** [comment]
I don't see us switching to podman, I think we want to keep this dependency basic. People complain about docker as is.

**Comment by evverx** [comment]
> I don't see us switching to podman

Fair enough. Though I think it would be great if OSS-Fuzz supported `podman` out of the box as well. It was kind of rejected in https://github.com/google/oss-fuzz/pull/4774 but I think it's worth revisiting because `podman` is much better integrated into RPM-based distributions at least.

**Comment by jonathanmetzman** [comment]
Related: https://github.com/google/oss-fuzz/issues/9299
